changelog/Microsoft.FileServices.json

```
@@ -2104,6 +2104,56 @@
         "CreatedDateTime": "2021-11-09T20:00:00.000Z",
         "WorkloadArea": "Files",
         "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "97b40989-8561-4ba0-a28c-24afb01fbeff",
+          "ApiChange": "Resource",
```

ApiChange: should be Relationship.

Was this automatically generated? If so, we should report this error.

---

Yes, this was automatically generated